@navikt/ds-react/src/link-panel/LinkPanel.tsx

```diff
@@ -16,39 +17,48 @@ export interface LinkPanelProps {
 }
 
 const LinkPanel: LinkPanelType = forwardRef(
-  (
-    {
-      children,
-      component: Component = "a",
-      border = true,
-      noHeading = false,
-      className,
-      ...rest
-    },
-    ref
-  ) => {
+  ({ children, component = "a", border = true, className, ...rest }, ref) => {
+    console.log(children, typeof children);
     return (
-      <Component
+      <Panel
+        component={component}
+        border={border}
         ref={ref}
-        className={cl("navds-link-panel", className, {
-          "navds-link-panel--border": border,
-        })}
+        className={cl("navds-link-panel", className)}
         {...rest}
       >
-        <span
-          className={cl("navds-link-panel__content", {
-            "navds-link-panel__heading": !noHeading,
-          })}
-        >
-          {children}
-        </span>
+        {typeof children === "string" ? (
```

Comment:
Denne sjekken kan muligens virke litt for magisk for folk?

Reply:
Er kanskje litt magisk, men tenker det fungerer greit hvis vi dokumenterer det godt. Fortrekker dette over en egen prop for å sette på link-styling